cask.el

```diff
@@ -730,7 +730,7 @@ If DEEP is t, all dependencies recursively will be returned."
            (-map
             (lambda (dependency)
               (list (cask-dependency-name dependency)
-                    (cask-dependency-version dependency)))
+                    (or (cask-dependency-version dependency) "0")))
```

Comment:
@lunaryorn I think version is more important if it is a package as you say, not so much for Emacs configuration. I think Cask should probably enforce version, as it is the "correct" way of doing it. If your package does not require a specific version, then that package should specify version "0" as a dependency version requirement.